scripts/max-loader.js

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env node
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+const async = require('async');
+const fs = require('fs');
+const path = require('path');
+const request = require('request');
+const requestProgress = require('request-progress');
+const mkdirp = require('mkdirp');
+const cp = require('child_process');
+const CronJob = require('cron').CronJob;
+
+var target = path.join(process.cwd(), '../db');
+
+// create db folder
+mkdirp.sync(target);
+
+log('Downloading to %s', target);
+
+// import the list of files to download
+var sources = require('../sources.json');
```

Comment:
Do you mean it should let you configure the name/path of the sources.json file, or configure the files that need to be downloaded? The latter is already available. 
is there a use case for the former?